op-tests/test-blspy-verify.txt

```diff
@@ -0,0 +1,32 @@
+; This file was generated by tools/generate-bls-tests.py
+
+bls_verify ((0xa4a108e5dabd6d6fde629fe9515cf397ab2a2d44c5740d39525aa7f5230df38d6753cb7a0683a75e2f4f3d82fd46cdb0 . 0xdb16a69196ce95ba4e04d0022f66b25b8f) (0xb66837657cd1564bb04e9acb1efd96b53ef90502bb000e6ca44da7e07f655e6fb50606f3cc47fd936ba4108ccf592625 . 0x40779f85fac1529f4eebe90c68ed3d3389d15111a797fdf9434045841b3a375310d5) (0x8f3eb0208a3f0808df538ebc1f582d01c62baabe12842171f7d6c01f9b31f7c9d386190d9349d53378998d0162c7ecb2 . 0xff27e2213e1faf566c4565ffede6bc89ec96bf89eaf4dbf343f01dfa7f55bdcad87845) (0xb066ce9e6484fe574ed55003449fae4d08652f85de45d718c2397c667cd80ed5571458d58c7d28b101ab1ef986973e77 . 0x5720658a115091d4f50561fb82f84b75a0f6ac41a808f0) (0xa73e3a434a117bed5c0eab24dce24b2e03bc75254b878900eb01f98cf389329cdd7d83cd8033475ff1ec6b2e10abbb09 . 0x31e471716c9f23491d) (0xb756ecad886b575d2c95014c20bcd9b52f1f72e3b5fbc657bacf1e72be707f9aaf76a9156a1628b99d873036beecec10 . 0x61e43e08f2) (0xae0154514ac83b8efea5b2eee425fc41ead032bb04b8a977eff85e80f9dd8940833c0d7d85921bf18fe84f639f417cca . 0x724dbefe2f80c6c12bd19cb407d847) ) 0x8ded853eb71f6015bc2c338cc2e2f2540e1d4cdc7d662965cd8827d6482e321e94ebd2cd299f77063336a531e294d879047d7017b93b232ce95a26e52cef34a1120aa4a3ab256242044d33e15b5430ff58612102bc80ef195a3adb4477cb7e1b => 1 | 36664624
```

Comment:
Here, I think the signature should come first so the `varargs` can come last. It would Probably simplify the rust implementation too.

Oh wait, I guess it's not actually `varargs`... it looks like it's a single argument that's a list.

Here again I wonder if we want the zipping of `g1` and `g2` to be done in the VM rather than clvm. We need to come up with a use case so we can check.

Reply:
yeah, I think I'm leaning towards varargs rather than a list of pairs. both for `bls_pairing_identity` and `bls_verify`